### PR TITLE
Start running Python 3.6 on Travis again

### PR DESCRIPTION
**Pull request by gsnedders**
Note we already have a Travis job for this, it's just currently doing nothing. (Well, that's untrue: it's creating the VM and everything, and spending about 90 seconds to achieve absolutely nothing.)

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/8523)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-11-30 17:08:03
Finished: 2017-11-30 17:23:04





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21299)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/309603755)

File: tools/ci/ci_tools_unittest.sh [hunk 1]
```diff
@@ -15,10 +15,8 @@ else
 fi
 
 if [[ $(./wpt test-jobs --includes wptrunner_unittest; echo $?) -eq 0 ]]; then
-    if [ $TOXENV == "py27" ] || [ $TOXENV == "pypy" ]; then
-        cd tools/wptrunner
-        tox
-    fi
+    cd tools/wptrunner
```
Comment (gsnedders) [inline tools/ci/ci_tools_unittest.sh]: Uh, this is wptrunner only. We should be running Py3.6 already? Hmm, how did it get so broken anyway?

**Comment by gsnedders** [comment]
So yeah, this PR is half wrong. But why is <https://travis-ci.org/w3c/web-platform-tests/jobs/309588422> not running anything? Hmm.

**Comment by gsnedders** [comment]
Ah, `before_install.sh` is failing, and therefore `RUN_JOBS` is never set and therefore everything goes up in flames.

**Comment by gsnedders** [comment]
But it's not returning a non-zero status code? Hmm.

**Comment by gsnedders** [comment]
Okay, this PR fixes the current issue, but the underlying issue why this didn't get caught before is https://github.com/w3c/web-platform-tests/issues/8525 (tl;dr: we've been throwing a Python exception in `before_install.sh` and ignoring it for a while).

**Review by jgraham — APPROVED** [review]

**Pull request merged**